site/docs/v1/tech/reference/cookies.adoc

```diff
@@ -0,0 +1,108 @@
+---
+layout: doc
+title: Hosted Login Pages Cookies
+description: A list of cookies set by the FusionAuth Hosted Login pages
+navcategory: admin
+---
+
+== Overview
```

Comment:
If I omit cookies that are used by the admin UI only, and omit transient cookies we are left with:

```
fusionauth.known-device
fusionauth.locale
fusionauth.pkce-verifier
fusionauth.remember-device
fusionauth.sso
fusionauth.timezone
fusionauth.trusted-device
fusionauth.trust_c
fusionauth.trust_t
fusionauth.trust
```